### DISCUSSION TITLE
Official statement from Turing Smart Screen

### DISCUSSION THREAD

**Discussion by EmperG** [mathoudebine/turing-smart-screen-python#115] [General]
Hi,mathoudebine
    Thank you for your interest in our products，we appreciate your job.
    However, we have to point out that we only have two hardware versions(version A), the rest are knockoffs.
   We noticed that many e-commerce websites are selling version b (XuanFang smart screen), and many customers thought it was our product, so they were fooled into buying it. We hope you can clarify this in the hardware introduction.
You can also communicate with us on our official website forum(www.turzx.com), we welcome this attempt at cross-platform.
      We currently have 3.5-inch and 5-inch products on sale, and soon 2.1-inch round screen and 8.8-inch long screen will be on the shelves. Products that we co-produce with many computer hardware brands will also be sold in 2023
     At last,We will be happy to provide you with the relevant hardware communication codes.

**Comment by mathoudebine** [comment]
Hi @EmperG 
Thank you for this clarification, I will update the documentation accordingly.
I started this project as a hobby and it attracted many people over the time!
I think there is a real community interested in your products for many use (system monitoring, smart home, bitcoin tracker, etc.)
I look forward for the new products and would be happy to integrate them to this project
Thanks for your support

**Comment by mathoudebine** [comment]
I pinned your message in the Discussion space of this project, so that everyone can see it easier

**Comment by RussNelson** [comment]
Hooray! I have long maintained that hardware manufacturers should cooperate with open source software authors by publishing documentation.

http://crynwr.com/on-being-proprietary.html

**Comment by alexwbaule** [comment]
HI @EmperG , can the themes been open to be copy to been used here ? Can you help it with this ?

**Reply by mathoudebine** [reply]
Hey @alexwbaule if you have `.data` theme files that you like from the official software, you can at least extract the background with a tool included in this project: https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-%3A-themes#import-a-theme-from-turing--xuanfang-windows-app
Unfortunately for now there is no option to convert `.data` themes to `theme.yaml` themes for this project

**Comment by alexwbaule** [comment]
Hi @mathoudebine  , i used the tool , and extract all backgrounds, but the elements position its hard to find the correct one. Im trying to figure out how to extract them too.

I build a PR to your project, add 5 inches devices support (kind of initial support, im using now )
